test/helpers/events.js

```diff
@@ -18,7 +18,7 @@ const fireCustomEvent = function (el, event, data) {
 	}));
 };
 
-module.exports = {
+export default {
```

Comment:
this one doesn't have both

Reply:
If they don't have both, babel-plugin-add-module-exports will add it for the CJS compat.
Did I get this wrong? Were we adding both in order to remove the use of babel-plugin-add-module-exports?